core/src/main/java/org/apache/iceberg/hadoop/HadoopTables.java

```diff
@@ -303,7 +303,8 @@ public Table create() {
       }
 
       Map<String, String> properties = propertiesBuilder.build();
-      TableMetadata metadata = tableMetadata(schema, spec, sortOrder, properties, location);
+
+      TableMetadata metadata = TableMetadata.newTableMetadata(schema, spec, sortOrder, location, properties);
```

Comment:
Is it necessary to change this file? Doesn't `tableMetadata` call `newTableMetadata`?

Reply:
My bad, I'll revert this change.